src/components/Team/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './style.less';
+import TeamCard from '../TeamCard';
+import emily from "assets/emily.png";
+import person from "assets/person.svg";
+
+const Team: React.FC = () => {
+  return (
+    <div className="Team">
+      <h2>Our Team <img alt="person" src={person}></img></h2>
+      <div className="teamCardContainer">
+        <TeamCard member="Emily" role="Supreme Leader" pic={emily}/>
```

Comment:
Could you organize it so that the team info is in a JSON file for member, role, and pics? Then the file can be parsed through - like https://github.com/acmucsd/main-website/tree/master/src/pages/about-us